app/src/state/rangeInput.tsx

```diff
@@ -2,18 +2,22 @@ import { Atom, PrimitiveAtom, atom } from "jotai";
 
 export type RangeInputMolecule = {
   name: string;
-  minAtom: PrimitiveAtom<string>;
-  maxAtom: PrimitiveAtom<string>;
+  minAtom: PrimitiveAtom<number>;
+  maxAtom: PrimitiveAtom<number>;
+  sliderMinAtom: PrimitiveAtom<number>;
+  sliderMaxAtom: PrimitiveAtom<number>;
```

Comment:
#### Should Fix
I agree that we need to add slider min & slider max to `RangeInputMolecule`, but I think we need to make some adjustments to naming & typing.

I think it will be better if we can rename `minAtom` to `minInputAtom` and `maxAtom` to `maxInputAtom` so that it is easier to distinguish what's actual minimum & what's user input.

`sliderMinAtom` & `sliderMaxAtom` do not need to be atoms because they are constant values.

So here's what I suggest:
- `minAtom: PrimitiveAtom<number>` -> `minInputAtom: PrimitiveAtom<number>`
- `maxAtom: PrimitiveAtom<number>` -> `maxInputAtom: PrimitiveAtom<number>`
- `sliderMinAtom: PrimitiveAtom<number>` -> `min: number`
- `sliderMaxAtom: PrimitiveAtom<number>` -> `max: number`